wot-wg-2023-draft.html

```diff
@@ -210,38 +210,50 @@ <h2>Scope</h2>
 
         <dl>
-          <dt><strong>Update Existing Specifications</strong>:</dt>
-          <dd>The WG will add new features and address issues discovered during implementation of already published specifications. 
-            Unlike the last charter, these would not be limited to backward-compatible updates.
+          <dt>
+		<strong>Support New Use Cases</strong>:
+          </dt>
+          <dd>Address the requirements of new use cases, such as Digital Twins, scalable events, 
+	    and interworking with edge and cloud systems.
+	    The WG intends to define canonical TDs, add a mechanism to sign TDs, 
+            define how to manage versions and how to validate TDs.
           </dd>
-          <dt><strong>Support WoT Interoperability</strong>:</dt>
+	  <dt>
+		<strong>Update Existing Specifications</strong>:
+	  </dt>
+          <dd>The WG will add new features and address issues discovered during implementation of the current set of specifications. 
+            These are not be limited to backward-compatible updates, however great care will be taken to minimize 
+	    the impact on existing implementations.
+          </dd>
+          <dt>
+		<strong>Support WoT Interoperability</strong>:
+	  </dt>
           <dd>The WG will improve out-of-the-box interoperability and enable the integration of WoT into other ecosystems and communities. 
-            Thus, the WG will define core binding and profiling mechanisms, specify additional protocol and payload bindings, and define appropriate profiles.
+            Thus, the WG will define core binding and profiling mechanisms, specify additional protocol and payload bindings, 
+            and define appropriate profiles.
           </dd>
           <dt>
-            <strong>Improve Management of TDs</strong>:
+          	<strong>Improve Management of TDs</strong>:
           </dt>
           <dd>
             Additional mechanisms around for the management of TDs are needed to improve workflows that use them. 
             Thus, the WG will update the Discovery specification with additional features, including geolocation. 
-            In other specifications, the WG intends to add a mechanism to sign TDs, define how to manage versions of TDs and define how to
-             parse, validate, consume and lint TDs.
           </dd>
           <dt>
-            <strong>Increase Descriptiveness of TDs</strong>:
+            	<strong>Increase Descriptiveness of TDs</strong>:
           </dt>
           <dd>
-            The WG will work on increasing the descriptive power of TDs to allow using them in more use cases. This includes
-            describing dynamic resources, historical values (timeseries), repeating payloads, both static and dynamic
+            The WG will work on increasing the descriptive power of TDs to allow using them in wider use cases. This includes
+            describing dynamic resources, historical values (timeseries), describing payloads, both static and dynamic
```

Comment:
we are already describing payloads so I would prefer to keep the previous wording

Reply:
There was a problem with describing event payloads - we discussed in the Profile TF some time ago.
I don't understand the use cases (and proposed solution) for repeating payloads.

Reply:
Agree with Ege.  There are some good ideas here, but mixed in with other things that I don't agree with, and it's too hard to address the things that need fixing with suggested changes.